CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing to the Reactive Streams Project
+
+The Reactive Streams project welcomes contributions from anybody who wants to participate in moving this initiative forward. All code or documentation that is contributed will have to be covered by a waiver of all copyrights and other rights as detailed by the LICENSE and COPYING files at each repository root, the rationale for this is that the APIs defined by this project shall be freely implementable and usable by everyone.
+
+## Copyright Statement
+
+The aforementioned waiver of copyrights and other rights is represented by the addition of a line to the file [CopyrightWaivers.txt](https://github.com/reactive-streams/reactive-streams-jvm/blob/master/CopyrightWaivers.txt). For a pull request to be considered every contributor must have signed the copyright statement in this way; this may be included within that same pull request.
+
+## Gatekeepers
+
+To ensure consistent development of Reactive Streams towards their goal, a group of gatekeepers is defined:
+
+* Kaazing Corp., currently represented by Todd Montgomery (@tmontgomery)
+* Netflix Inc., currently represented by Ben Christensen (@benjchristensen)
+* Pivotal Software Inc., currently represented by Jon Brisbin (@jbrisbin) and Stéphane Maldini (@smaldini)
+* Red Hat Inc., currently represented by Tim Fox (@purplefox) and Norman Maurer (@normanmaurer)
+* Twitter Inc., currently represented by Marius Eriksen (@mariusaeriksen)
+* Typesafe Inc., currently represented by Viktor Klang (@viktorklang) and Roland Kuhn (@rkuhn)
+
```

Comment:
This list is somewhat outdated now; shall I write up the changes needed or we leave as is? cc @viktorklang